nodejs/repositories.bzl

```diff
@@ -236,8 +233,9 @@ def _prepare_node(repository_ctx):
     node_entry = "bin/node%s" % entry_ext
     npm_entry = "bin/npm%s" % entry_ext
 
-    node_bin_relative = paths.relativize(node_bin, "bin")
-    npm_script_relative = paths.relativize(npm_script, "bin")
+    node_bin_relative = node_bin[len("bin/"):]
```

Comment:
perhaps check if the paths start with `bin/` and fail if they don't as a defence in depth?